pkgs/by-name/ca/catalyst/package.nix

```
@@ -0,0 +1,86 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  appimageTools,
+  makeWrapper,
+  electron_33,
+  electronPackage ? electron_33,
+  asar,
+}:
+
+let
+  electron = electronPackage;
+in
+stdenv.mkDerivation rec {
+  pname = "catalyst";
+  version = "3.9.4";
+
+  src = fetchurl {
+    url = "https://github.com/CatalystDevOrg/Catalyst/releases/download/v${version}/catalyst-${version}.AppImage";
+    hash = "sha256-6t1RAxmRc/1fAQT4Qnd42kh3cxgRZr74k8gwebTb0Ic=";
+    name = "${pname}-${version}.AppImage";
+  };
+
+  buildInputs = [
+    asar
+  ];
+
+  appimageContents = appimageTools.extractType2 {
+    name = "${pname}-${version}";
+    inherit src;
+  };
+
+  dontUnpack = true;
+  dontConfigure = true;
+  dontBuild = true;
+
+  nativeBuildInputs = [ makeWrapper ];
```

```suggestion
  nativeBuildInputs = [ makeWrapper asar ];
```
or on two lines, depending on nixfmt